### DISCUSSION TITLE
API does not give out basic information about invoices onchain details.  [500k sat bounty]

### DISCUSSION THREAD

**Discussion by Transisto** [btcpayserver/btcpayserver#2797] [General]
The most important missing info for us is the receiving address and TXIDs of incoming transactions.

Without this it's making it harder to cross validate payment with other services.

We currently have to webscape the invoices page from admin panel to get this.

Overall I think everything that is visible from the invoice page should be accessible via the API.

https://docs.btcpayserver.org/API/Greenfield/v1/#operation/Invoices_GetInvoice
500k sat bounty for allowing us to extract Address and related TXID from the API.

**Comment by Kukks** [comment]
https://docs.btcpayserver.org/API/Greenfield/v1/#operation/Invoices_GetInvoicePaymentMethods
has all the info you are asking for

**Reply by Transisto** [reply]
Thanks, made a fool of myself.
